### PR TITLE
feat: Add ability to configure python

### PR DESCRIPTION
**Pull request by jpopesculian**
This adds the ability to select the python version for the virtual environment created by the CLI. It emits a warning if the python version for the virtual environment is already configured and a new one is passed in. It also selects the aqora bin inside of the virtual environment if its available.

### CODE REVIEW

**Comment by jpopesculian** [comment]
Related also to https://github.com/aqora-io/platform/issues/318

**Review by volgar1x — COMMENTED** [review]
It's a great feature, yet confusing since you can `aqora info --python python3.11` and get printed `INFO Python 3.12.4`. Maybe we should allow the `--python` flag only when creating/installing a project?

**Comment by jpopesculian** [comment]
fixes #72

**Comment by Angel-Dijoux** [comment]
I would love try this PR but I have a issue with the build, that I assume come from my computer, it's look like this : 

```bash
......
win/lib/librustc_std_workspace_alloc-6ef0176aaa60ff0c.rlib" "/Users/angeldijoux/.rustup/toolchains/stable-aarch64-apple-darwin/lib/rustlib/aarch64-apple-darwin/lib/libminiz_oxide-1e9f0e423eed4f7c.rlib" "/Users/angeldijoux/.rustup/toolchains/stable-aarch64-apple-darwin/lib/rustlib/aarch64-apple-darwin/lib/libadler-263f3ba6f4d2645b.rlib" "/Users/angeldijoux/.rustup/toolchains/stable-aarch64-apple-darwin/lib/rustlib/aarch64-apple-darwin/lib/libunwind-85e43ed53a81d633.rlib" "/Users/angeldijoux/.rustup/toolchains/stable-aarch64-apple-darwin/lib/rustlib/aarch64-apple-darwin/lib/libcfg_if-279824e18f4fd20b.rlib" "/Users/angeldijoux/.rustup/toolchains/stable-aarch64-apple-darwin/lib/rustlib/aarch64-apple-darwin/lib/liblibc-b7ead8c5aa11dde6.rlib" "/Users/angeldijoux/.rustup/toolchains/stable-aarch64-apple-darwin/lib/rustlib/aarch64-apple-darwin/lib/liballoc-37d126161ada8ba6.rlib" "/Users/angeldijoux/.rustup/toolchains/stable-aarch64-apple-darwin/lib/rustlib/aarch64-apple-darwin/lib/librustc_std_workspace_core-c7113231a51981ef.rlib" "/Users/angeldijoux/.rustup/toolchains/stable-aarch64-apple-darwin/lib/rustlib/aarch64-apple-darwin/lib/libcore-0e8873809402687b.rlib" "/Users/angeldijoux/.rustup/toolchains/stable-aarch64-apple-darwin/lib/rustlib/aarch64-apple-darwin/lib/libcompiler_builtins-c3f3955ff7203236.rlib" "-framework" "SystemConfiguration" "-framework" "CoreFoundation" "-lpython3.9" "-liconv" "-lSystem" "-lc" "-lm" "-L" "/Users/angeldijoux/.rustup/toolchains/stable-aarch64-apple-darwin/lib/rustlib/aarch64-apple-darwin/lib" "-o" "/Users/angeldijoux/Documents/aqora/cli/target/debug/deps/libaqora_cli.dylib" "-Wl,-dead_strip" "-dynamiclib" "-Wl,-dylib" "-nodefaultlibs"
  = note: ld: warning: search path '/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib' not found
          ld: library 'python3.9' not found
          clang: error: linker command failed with exit code 1 (use -v to see invocation)
          

error: could not compile `aqora` (lib) due to 1 previous error        
```

I believe that not come from this PR, because is still the same with main branch /:

File: src/commands/global_args.rs [hunk 1]
```diff
@@ -27,6 +27,10 @@ pub struct GlobalArgs {
     pub project: PathBuf,
     #[arg(long, global = true)]
     pub uv: Option<PathBuf>,
+    #[arg(long, global = true)]
+    pub python: Option<String>,
+    #[arg(long, global = true, default_value = "false")]
+    pub ignore_venv_aqora: bool,
```
Comment (Angel-Dijoux) [inline src/commands/global_args.rs:33]: This is very cool!

**Review by Angel-Dijoux — APPROVED** [review]
The code is very great, let me setup a virtual machine and I test it inside this instead of my computer!

**Pull request merged**